deployments/kubehound/notebook/LowHangingFruit-ContainerEscape.ipynb

```diff
@@ -0,0 +1,550 @@
+    "%%graph_notebook_vis_options\n",
+    "{\n",
+    "  \"edges\": {\n",
+    "    \"smooth\": {\n",
+    "      \"enabled\": true,\n",
+    "      \"type\": \"dynamic\"\n",
+    "    },\n",
+    "    \"arrows\": {\n",
+    "      \"to\": {\n",
+    "        \"enabled\": true,\n",
+    "        \"type\": \"arrow\"\n",
+    "      }\n",
+    "    }\n",
+    "  }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "frozen": false,
+    "tags": [
+     "unsafe_output"
+    ]
+   },
+   "source": [
+    "# Initial Setup\n",
+    "\n",
+    "## Get a view of all Ingested Cluster\n",
+    "\n",
+    "Retrieve all the current cluster ingested in KubeHound with the associated runID with the number of nodes. This numbers can be used to get a clue of the size of the cluster and also identify if an ingestion did not complete."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2024-07-24T14:46:28.273187Z",
+     "iopub.status.busy": "2024-07-24T14:46:28.272852Z",
+     "iopub.status.idle": "2024-07-24T14:46:28.625859Z",
+     "shell.execute_reply": "2024-07-24T14:46:28.625126Z",
+     "shell.execute_reply.started": "2024-07-24T14:46:28.273156Z"
+    },
+    "frozen": false,
+    "tags": [
+     "safe_output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "%%gremlin -d class -g critical -le 50 -p inv,oute\n",
+    "\n",
+    "kh.nodes()\n",
+    "    .groupCount()\n",
+    "    .by(project('cluster','runID')\n",
+    "         .by('cluster').by('runID'))\n",
+    "    .unfold()\n",
+    "    .limit(1000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "frozen": false,
+    "tags": [
+     "unsafe_output"
+    ]
+   },
+   "source": [
+    "## Setting your run_id/cluster\n",
+    "\n",
+    "Set which runID you want to use. The variable are being shared with all users of the instance, so we advise to make a uniq string for your user `runID_yourid` to avoid any conflict."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "frozen": false,
+    "tags": [
+     "safe_output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "%%gremlin -d class -g critical -le 50 -p inv,oute\n",
+    "\n",
+    "graph.variables()\n",
+    "    .set('runID_yourid','01htdgjj34mcmrrksw4bjy2e94')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "frozen": false,
+    "tags": [
+     "unsafe_output"
+    ]
+   },
+   "source": [
+    "# Container escapes\n",
+    "\n",
+    "List all containers which are vulnerable to container escape to the node. \n",
+    "\n",
+    "## Identify the vulnerable containers\n",
+    "\n",
+    "The goal of this list is to identify which image is being vulnerable to container escape. It will list all the containers remove duplicate entry that share the same labels `namespace`, `app`, `team` and `image`."
```

Comment:
```suggestion
    "The goal of this list is to identify images vulnerable to container escape. It will list all the containers and remove duplicate entry that share the same `namespace`, `app`, `team` and `image` labels."
```